scripts/quicklink-to-project.sh

```diff
@@ -0,0 +1,22 @@
+if [ -z "$1" ] && [ -z "$PROJECT_PATH" ]; then
+  echo "Must supply a project name or set the PROJECT_PATH env"
+fi
+
+PROJECT=${PROJECT_PATH:-../$1}
+
+if [ ! -d "$PROJECT" ]; then
+  echo "$(realpath $PROJECT) doesn't exist, ensure project name is correct or PROJECT_PATH is pointing to a valid directory"
+  exit 1
+fi
+
+cd $PROJECT
+
+# Does package have integrate command?
+cat package.json | jq -e .scripts.integrate &>2
+
+if [ $? -ne 0 ]; then
+  echo "$PROJECT doesn't have integrate npm script, ensure it's valid"
```

Comment:
Maybe reference something that would make it valid?

Reply:
Like pointing to another project?

Reply:
Or something -- a random dev won't know what "ensure it's valid' means.

Reply:
And/or perhaps re-word it so people know to look at the `scripts` key of `package.json`? I’m not sure “npm scripts” is something everybody will immediately understand.

Reply:
I updated the wording a little bit. I'm gonna write a blog post and some more detailed documentation on this process this week so I'll follow up once that's done.